var/spack/repos/builtin/packages/e3sm-kernels/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+import os
+
+
+class E3smKernels(MakefilePackage):
+    """
+    Climate kernels for Co-design that originate from the Energy
+    Exascale Earth System Model (E3SM).
+    """
+
+    homepage = "https://github.com/e3SM-Project/codesign-kernels"
+    url = "https://github.com/E3SM-Project/codesign-kernels/archive/refs/tags/v1.0.tar.gz"
+    git = "https://github.com/E3SM-Project/codesign-kernels.git"
+
+    maintainers = ["sarats", "philipwjones"]
```

Comment:
Neither of these accounts seem to be accessible here, at least I can't include them with `@<account>`. Would you care to be the maintainer *of this package*?

Reply:
Hmm.  I wonder why `spackbot` can ping them.

Reply:
I grabbed them from the actual repo of the code itself. Let me ping a few other people to see if they are available before we fall back to that. Looks like I failed to run `spack style` correctly anyway and need to fix an import order

Reply:
Ummm...not sure what this is about? @sarats - do you know anything?

Reply:
The ECP Proxy Apps project are cleaning up and adding a few proxy apps to spack that were previously part of the catalog.

I forget what the official responsibility of a maintainer in spack is but mostly it boils down to taking a look at what people do and making sure it isn't too insane or wrong.

Reply:
That's correct.  Signing up to be a `maintainer` essentially means you will get "pinged" when someone submits a PR to change the package to give you a chance to review the changes.

Reply:
@philipwjones @sarats Let us know if you do not want to be included in the `maintainers` list so it can be updated before this PR is merged.